src/components/progress-report/workflow/transitions.ts

```diff
@@ -11,78 +11,106 @@ import {
 // Therefore, these should generally flow down.
 // "back transactions" should come before/above "forward transactions".
 
+/*
+  The following Roles are always notifed for each transition and are handled within
+  progress-report-workflow-notification.handler:
+    Role.ProjectManager,
+    Role.RegionalDirector,
+    Role.FieldOperationsDirector,
+*/
```

Comment:
I get that this is trying to be helpful, but this is a prime comment to become stale in the future, when that actual code changes and this doesn't.
I'd say something like
```ts
/**
 * There could be roles that are implicitly always notified.
 * @see import('./handlers/progress-report-workflow-notification.handler').rolesToAlwaysNotify
 */
```
The jsdoc syntax allows you to jump right to the defined list in your IDE.